src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocCheckParagraphOnPlainComment.java

```diff
@@ -6,15 +6,12 @@
 tagOrder = (default)@author, @deprecated, @exception, @param, @return, \
            @see, @serial, @serialData, @serialField, @since, @throws, @version
 
-
```

Comment:
These changes are  but not really necessary but fine because the input doesn't fire any violations. If it did, then you would have had to modify the expected string too. As of now, this is okay